protobuf/protoc-gen-cruxclient/example/route_guide.crux.client.pb.cc

```diff
@@ -9,6 +9,28 @@ namespace routeguide {
 
 RouteGuideClient::RouteGuideClient(const std::shared_ptr<RouteGuide::StubInterface>& stub) : mStub(stub) {}
 
+void RouteGuideClient::Invoke(const google::protobuf::Any& request_data, const std::string& method) const {
+  if (method == kRouteGuideGetFeature) {
+    routeguide::Point request;
+    if (!request.ParseFromString(request_data.value())) {
```

Comment:
Why do we `ParseFromString` is this not already in the correct format?
```suggestion
    if (!request_data.UnpackTo(&request)) {
```